webviz_config/templates/webviz_template.py.jinja2

```diff
@@ -128,45 +129,41 @@ else:
     {% endfor %}
     app.layout = html.Div(
         className="layoutWrapper", 
-        children=[html.Div(
-            children=[dcc.Location(
-                id='url', refresh=True),
-        html.Div(
-            className="sideWrapper",
-            children=[
-            {% for page in pages %}
-                dcc.Link(
-                    {%- if loop.first -%}
-                    "",
-                    id="logo",
-                    className="styledLogo",
-                    href="/",
-                    {%- else -%}
-                    "{{page.title}}",
-                    # We will create a webviz-core-components
-                    # component instead of styling dcc.Link's,
-                    # then we can more easily change className to
-                    # selectedButton for current page.
-                    className="styledButton",
-                    id="{{page.id}}",
-                    href="/{{page.id}}",
-                    {%- endif -%}
+        children=[
+            html.Div(
+                children=[
+                    dcc.Location(
```

Comment:
I think the `dcc.Location` can be removed? It was used in the old page navigation.

Reply:
`dcc.Location` is still used for reading the initial URL and loading the respective page (e.g. when you enter `https://webviz-subsurface-example.azurewebsites.net/well-log-viewer` it loads the `WellLogViewer`).